From 1fc676b74ad3c95670496bcabed0aa2837fc3e8c Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Fri, 23 Mar 2018 21:20:39 -0400
Subject: [PATCH] find libm correctly

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 340caa0..36bd4a1 100644
--- a/meson.build
+++ b/meson.build
@@ -67,7 +67,7 @@ libgd = subproject(
 )
 libgd_dep = libgd.get_variable('libgd_dep')
 
-libm = cc.find_library('libm')
+libm = cc.find_library('m')
 
 gail = dependency('gail-3.0')
 if get_option('extensions')
-- 
2.16.2

