From 71f747ad793aeab16ef49e38c241cfcb2b50ec79 Mon Sep 17 00:00:00 2001
From: brl <bruce@subgraph.com>
Date: Wed, 6 Dec 2017 20:42:29 -0500
Subject: [PATCH] prefix needed for ncurses check

---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 09248d4..1e28505 100644
--- a/configure.ac
+++ b/configure.ac
@@ -414,8 +414,8 @@ ncurses_found=no
 dnl Firstly check for lib${curses_lib_name}.
 AC_CHECK_LIB(${curses_lib_name}, initscr,
 	[LIBS="$LIBS -l${curses_lib_name}"
-	if test x$vifm_cv_curses = x/usr -a -d /usr/include/${curses_lib_name}; then
-		CPPFLAGS="$CPPFLAGS -I/usr/include/${curses_lib_name}"
+	if test x$vifm_cv_curses = x/usr -a -d ${prefix}/usr/include/${curses_lib_name}; then
+		CPPFLAGS="$CPPFLAGS -I${prefix}/usr/include/${curses_lib_name}"
 	fi
 	ncurses_found=yes]
 	AC_SEARCH_LIBS([curs_set], [tinfo],,
-- 
2.14.3

