From 63162a620053f3d21a5610be50e66a72595ebf09 Mon Sep 17 00:00:00 2001
From: brl <bruce@subgraph.com>
Date: Tue, 12 Dec 2017 22:33:08 -0500
Subject: [PATCH] remove check for zenity

---
 configure.ac | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index 89c15e2..5b3414e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -390,11 +390,6 @@ fi
 
 AM_CONDITIONAL(HAVE_SM, test "$found_sm" = "yes")
 
-AC_PATH_PROG(ZENITY, zenity, no)
-if test x"$ZENITY" = xno; then
-  AC_MSG_ERROR([zenity not found in your path - needed for dialogs])
-fi
-
 AC_ARG_ENABLE(debug,
 	[  --enable-debug		enable debugging],,
 	enable_debug=no)
-- 
2.14.3

