From 39d14e08905e1ee6aed5f9659c13e9c9bb02a1dd Mon Sep 17 00:00:00 2001
From: brl <bruce@subgraph.com>
Date: Tue, 19 Dec 2017 14:11:16 -0500
Subject: [PATCH 3/5] disable running tests

---
 shell/Makefile.am | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/shell/Makefile.am b/shell/Makefile.am
index de3e53d..54fbdf8 100644
--- a/shell/Makefile.am
+++ b/shell/Makefile.am
@@ -163,10 +163,10 @@ EXTRA_DIST =					\
 CLEANFILES = $(BUILT_SOURCES) $(completion_DATA) $(servicefile_DATA)
 DISTCLEANFILES = gnome-control-center.desktop gnome-control-center.desktop.in
 
-TEST_PROGS += test-hostname
-noinst_PROGRAMS += test-hostname
-test_hostname_SOURCES = hostname-helper.c hostname-helper.h test-hostname.c
-test_hostname_LDADD = $(PANEL_LIBS) $(INFO_PANEL_LIBS)
+#TEST_PROGS += test-hostname
+#noinst_PROGRAMS += test-hostname
+#test_hostname_SOURCES = hostname-helper.c hostname-helper.h test-hostname.c
+#test_hostname_LDADD = $(PANEL_LIBS) $(INFO_PANEL_LIBS)
 
 EXTRA_DIST += hostnames-test.txt ssids-test.txt
 
-- 
2.14.3

