From 3f4c934ca6c1bdfc2c49158da254771cd78586cd Mon Sep 17 00:00:00 2001
From: brl <bruce@subgraph.com>
Date: Tue, 19 Dec 2017 14:14:37 -0500
Subject: [PATCH 4/5] remove unwanted panel dependencies

---
 configure.ac | 80 ++++++++++++++++++++++++++++--------------------------------
 1 file changed, 37 insertions(+), 43 deletions(-)

diff --git a/configure.ac b/configure.ac
index 42ddc62..8f53b86 100644
--- a/configure.ac
+++ b/configure.ac
@@ -121,9 +121,7 @@ PKG_CHECK_MODULES(LIBLANGUAGE, $COMMON_MODULES gnome-desktop-3.0 fontconfig)
 PKG_CHECK_MODULES(LIBSHORTCUTS, $COMMON_MODULES x11)
 PKG_CHECK_MODULES(SHELL, $COMMON_MODULES x11 polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION)
 PKG_CHECK_MODULES(BACKGROUND_PANEL, $COMMON_MODULES cairo-gobject libxml-2.0 gnome-desktop-3.0
-                  gdk-pixbuf-2.0 >= $GDKPIXBUF_REQUIRED_VERSION
-                  goa-1.0 >= $GOA_REQUIRED_VERSION
-                  grilo-0.3 >= $GRILO_REQUIRED_VERSION)
+                  gdk-pixbuf-2.0 >= $GDKPIXBUF_REQUIRED_VERSION)
 PKG_CHECK_MODULES(DATETIME_PANEL, $COMMON_MODULES
                   gnome-desktop-3.0 >= $GNOME_DESKTOP_REQUIRED_VERSION
                   polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION
@@ -141,15 +139,11 @@ PKG_CHECK_MODULES(MOUSE_PANEL, $COMMON_MODULES xi >= 1.2
 PKG_CHECK_MODULES(NETWORK_PANEL, $COMMON_MODULES gmodule-2.0
                   polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION)
 PKG_CHECK_MODULES(NOTIFICATIONS_PANEL, $COMMON_MODULES)
-PKG_CHECK_MODULES(ONLINE_ACCOUNTS_PANEL, $COMMON_MODULES goa-1.0 goa-backend-1.0 >= $GOA_REQUIRED_VERSION)
 PKG_CHECK_MODULES(COLOR_PANEL, $COMMON_MODULES
                   colord >= $COLORD_REQUIRED_VERSION
                   colord-gtk >= $COLORD_GTK_REQUIRED_VERSION
                   libsoup-2.4
                   gnome-desktop-3.0 >= $GNOME_DESKTOP_REQUIRED_VERSION)
-PKG_CHECK_MODULES(PRINTERS_PANEL, $COMMON_MODULES
-                  polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION
-                  smbclient)
 PKG_CHECK_MODULES(PRIVACY_PANEL, $COMMON_MODULES)
 PKG_CHECK_MODULES(REGION_PANEL, $COMMON_MODULES
                   polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION
@@ -269,36 +263,36 @@ fi
 # Check for CUPS 1.4 or newer
 AC_PROG_SED
 
-AC_PATH_PROG(CUPS_CONFIG, cups-config)
+#AC_PATH_PROG(CUPS_CONFIG, cups-config)
 
-if test x$CUPS_CONFIG = x; then
-  AC_MSG_ERROR([cups-config not found])
-fi
+#if test x$CUPS_CONFIG = x; then
+#  AC_MSG_ERROR([cups-config not found])
+#fi
 
-CUPS_API_VERSION=`$CUPS_CONFIG --api-version`
-CUPS_API_MAJOR=`echo $ECHO_N $CUPS_API_VERSION | cut -d . -f 1`
-CUPS_API_MINOR=`echo $ECHO_N $CUPS_API_VERSION | cut -d . -f 2`
+#CUPS_API_VERSION=`$CUPS_CONFIG --api-version`
+#CUPS_API_MAJOR=`echo $ECHO_N $CUPS_API_VERSION | cut -d . -f 1`
+#CUPS_API_MINOR=`echo $ECHO_N $CUPS_API_VERSION | cut -d . -f 2`
 
-AC_CHECK_HEADERS([cups/cups.h cups/http.h cups/ipp.h cups/ppd.h],,
-                 AC_MSG_ERROR([CUPS headers not found]))
+#AC_CHECK_HEADERS([cups/cups.h cups/http.h cups/ipp.h cups/ppd.h],,
+#                 AC_MSG_ERROR([CUPS headers not found]))
 
-if ! test $CUPS_API_MAJOR -gt 1 -o \
-          $CUPS_API_MAJOR -eq 1 -a $CUPS_API_MINOR -ge 4 ; then
-  AC_MSG_ERROR([CUPS 1.4 or newer not found])
-fi
+#if ! test $CUPS_API_MAJOR -gt 1 -o \
+#          $CUPS_API_MAJOR -eq 1 -a $CUPS_API_MINOR -ge 4 ; then
+#  AC_MSG_ERROR([CUPS 1.4 or newer not found])
+#fi
 
 # https://bugzilla.gnome.org/show_bug.cgi?id=696766
-CUPS_CPPFLAGS=""
-if test $CUPS_API_MAJOR -gt 1 -o \
-        $CUPS_API_MAJOR -eq 1 -a $CUPS_API_MINOR -ge 6 ; then
-   CUPS_CPPFLAGS=-D_PPD_DEPRECATED=""
-fi
-
-CUPS_CFLAGS=`$CUPS_CONFIG --cflags | $SED -e 's/-O\w*//g' -e 's/-m\w*//g'`
-CUPS_LIBS=`$CUPS_CONFIG --libs`
-AC_SUBST(CUPS_CPPFLAGS)
-AC_SUBST(CUPS_CFLAGS)
-AC_SUBST(CUPS_LIBS)
+#CUPS_CPPFLAGS=""
+#if test $CUPS_API_MAJOR -gt 1 -o \
+#        $CUPS_API_MAJOR -eq 1 -a $CUPS_API_MINOR -ge 6 ; then
+#   CUPS_CPPFLAGS=-D_PPD_DEPRECATED=""
+#fi
+
+#CUPS_CFLAGS=`$CUPS_CONFIG --cflags | $SED -e 's/-O\w*//g' -e 's/-m\w*//g'`
+#CUPS_LIBS=`$CUPS_CONFIG --libs`
+#AC_SUBST(CUPS_CPPFLAGS)
+#AC_SUBST(CUPS_CFLAGS)
+#AC_SUBST(CUPS_LIBS)
 
 # Optional dependency for the user accounts panel
 AC_ARG_WITH([cheese],
@@ -336,21 +330,21 @@ fi
 AM_CONDITIONAL(BUILD_WACOM, [test x${have_wacom} = xyes])
 
 # Kerberos kerberos support
-AC_PATH_PROG(KRB5_CONFIG, krb5-config, no)
-if test "$KRB5_CONFIG" = "no"; then
-  AC_MSG_ERROR([krb5-config executable not found in your path - should be installed with the kerberos libraries])
-fi
+#AC_PATH_PROG(KRB5_CONFIG, krb5-config, no)
+#if test "$KRB5_CONFIG" = "no"; then
+#  AC_MSG_ERROR([krb5-config executable not found in your path - should be installed with the kerberos libraries])
+#fi
 
-AC_MSG_CHECKING(for krb5 libraries and flags)
-KRB5_CFLAGS="`$KRB5_CONFIG --cflags`"
-KRB5_LIBS="`$KRB5_CONFIG --libs`"
-AC_MSG_RESULT($KRB5_CFLAGS $KRB5_LIBS)
+#AC_MSG_CHECKING(for krb5 libraries and flags)
+#KRB5_CFLAGS="`$KRB5_CONFIG --cflags`"
+#KRB5_LIBS="`$KRB5_CONFIG --libs`"
+#AC_MSG_RESULT($KRB5_CFLAGS $KRB5_LIBS)
 
-AC_SUBST(KRB5_CFLAGS)
-AC_SUBST(KRB5_LIBS)
+#AC_SUBST(KRB5_CFLAGS)
+#AC_SUBST(KRB5_LIBS)
 
-USER_ACCOUNTS_PANEL_CFLAGS="$USER_ACCOUNTS_PANEL_CFLAGS $KRB5_CFLAGS"
-USER_ACCOUNTS_PANEL_LIBS="$USER_ACCOUNTS_PANEL_LIBS $KRB5_LIBS"
+#USER_ACCOUNTS_PANEL_CFLAGS="$USER_ACCOUNTS_PANEL_CFLAGS $KRB5_CFLAGS"
+#USER_ACCOUNTS_PANEL_LIBS="$USER_ACCOUNTS_PANEL_LIBS $KRB5_LIBS"
 
 dnl Check for info panel
 AC_ARG_WITH([gnome-session-libexecdir],
-- 
2.14.3

