From 78929b37ba1ee7408170299c9580840c9ca23663 Mon Sep 17 00:00:00 2001
From: brl <bruce@subgraph.com>
Date: Tue, 19 Dec 2017 14:15:45 -0500
Subject: [PATCH 5/5] fix gdesktop prefix

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 8f53b86..3f654fc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -169,7 +169,7 @@ PKG_CHECK_MODULES(REMOTE_LOGIN_HELPER, glib-2.0 >= $GLIB_REQUIRED_VERSION gio-2.
 PKG_CHECK_MODULES(GVC, gobject-2.0 libpulse libpulse-mainloop-glib)
 AM_CONDITIONAL(HAVE_INTROSPECTION, false)
 
-GDESKTOP_PREFIX=`$PKG_CONFIG --variable prefix gsettings-desktop-schemas`
+GDESKTOP_PREFIX=${PKG_CONFIG_SYSROOT_DIR}`$PKG_CONFIG --variable prefix gsettings-desktop-schemas`
 AC_SUBST(GDESKTOP_PREFIX)
 
 PKG_CHECK_MODULES(DEVICES, $COMMON_MODULES)
-- 
2.14.3

