--- a/src/meta/window.h	2018-03-12 18:45:19.000000000 +0000
+++ b/src/meta/window.h	2018-10-25 18:15:11.258297608 +0000
@@ -205,6 +205,7 @@
 guint       meta_window_get_stable_sequence (MetaWindow *window);
 guint32     meta_window_get_user_time (MetaWindow *window);
 int         meta_window_get_pid (MetaWindow *window);
+guint32     meta_window_get_client_pid (MetaWindow *window);
 const char *meta_window_get_client_machine (MetaWindow *window);
 gboolean    meta_window_is_remote (MetaWindow *window);
 gboolean    meta_window_is_attached_dialog (MetaWindow *window);
